File: array/c/Insert.c
#include<stdio.h>
int main(){
    int LA[7] = { 5,10,15,20,25}, N = 5, K = 3, item = 100;
    int J = N;
    while(J >= K){
        LA[J + 1] = LA[J];
        J--;
    }
    LA[K] = item;
    N = N + 1;
    int i;
    for(i = 0; i <= N; i++){
        printf("%d ", LA[i]);
    }
    return 0;
}

File: array/python/Insert.py
# Array Insert operations

number_array = [4, 5, 2, 7, 9, 13]
print("=====================Initial Array================")
for index, item in enumerate(number_array):
    print("Array=[", index, "]", item)

n=len(number_array)
k=3 # position of index to insert
item=15 # value to insert

while k>n:
    print("Invalid position")
    # k=int(input("Enter position to insert: "))

number_array.insert(k, item)
print("=====================Inserted Array================")
for index, item in enumerate(number_array):
    print("Array=[", index, "]", item)